src/installer/pkg/sfx/installers/dotnet-runtime-deps/dotnet-runtime-deps-debian.proj

```diff
@@ -5,7 +5,7 @@
 
   <ItemGroup>
     <LinuxPackageDependency Include="libc6;libgcc1;libgssapi-krb5-2;libstdc++6;zlib1g"/>
-    <LinuxPackageDependency Include="libssl1.0.0 | libssl1.0.2 | libssl1.1" />
+    <LinuxPackageDependency Include="libssl1.0.0 | libssl1.0.2 | libssl1.1 | libssl3" />
```

Comment:
At least for the build of .NET Runtime, the minimum requirement is libssl1.0.2, i.e. on Debian Jessie (v8), libssl1.0.1 is available in default packages, and it fails the build. Upgrading to 1.0.2 (from jessie-backports apt repository) fixes the build. I noticed it while working on [this](https://gist.github.com/am11/11b8f526747c5ad12c9a74e7765e14a4) script.

We may want to drop libssl1.0.0 from this list as well.

cc @bartonjs